README.md

```diff
@@ -1,21 +1,32 @@
+The Deploy Machine Learning Models to Azure action will deploy your model on [Azure Machine Learning](https://azure.microsoft.com/en-us/services/machine-learning/) using GitHub Actions.
+
+Get started today with a [free Azure account](https://azure.com/free/open-source)!
+
+This repository contains  GitHub Action for deploying Machine Learning Models to Azure Machine Learning and creates a real-time endpoint on the model to integrate models in other systems. The endpoint will be hosted either on an Azure Container Instance or on an Azure Kubernetes Service. 
+
 
 This GitHub Action also allows you to provide a python script that executes tests against the  Webservice endpoint after the model deplyoment has completed successfully. You can enable tests by setting the parameter `test_enabled` to true. In addition to that, you have to provide a python script (default `code/test/test.py`) which includes a function (default ` def main(webservice):`) that describes your tests that you want to execute against the service object. The python script gets the [webservice object](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.webservice(class)?view=azure-ml-py) injected. The action fails, if the test script fails.
 
-This action requires an Azure Machine Learning workspace to be created or attached to via the [aml-workspace](https://github.com/Azure/aml-workspace) action and an AKS cluster if you are planning to deploy your model to such a compute target. AKS clsuters can be managed via the [aml-compute](https://github.com/Azure/aml-compute) action.
 
-## Template repositories
+## Dependencies on other Github Actions
+* [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into Github Actions agent.
+* [aml-workspace](https://github.com/Azure/aml-workspace) This action requires an Azure Machine Learning workspace to be present. You can either create a new one or re-use an existing one using the action. 
+* [aml-registermodel]() Before deploying the model, you need this action to register the model with Azure Machine Learning and use its output in deploy action. 
+* [aml-compute](https://github.com/Azure/aml-compute) You don't need this if you want to host your endpoint on an ACI instance. But, if you want to host your endpoint on an AKS cluster, you can  managed the AKS Cluster via the action. 
 
-This action is one in a series of actions that can be used to setup an ML Ops process. Examples of these can be found at
-1. Simple example: [ml-template-azure](https://github.com/machine-learning-apps/ml-template-azure) and
-2. Comprehensive example: [aml-template](https://github.com/Azure/aml-template).

```

Comment:
I am basically talking about removing this part. I think we should keep this section, because this is crucial for the experience.

Reply:
Second thing is, only dependency today that exist between actions is on aml-workspace action. 
all other actions are independent, and integration tests show that clearly.
so ideally deploy action only depends upon aml-workspace

Reply:
Good point. You are right @ashishonce .

Reply:
I agree @marvinbuss. We don't need both, but it would be best to have at least one of these since we are essentially using that repo to show people how the actions can be used together.

Reply:
@awmatheson @ashishonce Can you please review the changed document? I will close this for now.